tasks/create_accounts.yml

```diff
@@ -23,7 +23,9 @@
     group: "{{ item.group | default(omit) }}"
     groups: "{{ (item.groups | default([])) | join(',') }}"
     home: "{{ item.home | default(omit) }}"
+    move_home : "{{ item.move_home | default(omit) }}"
```

Comment:
Hello,

Thanks for the proposal, can you fix the space before the colon please.